src/lib/get_graph_div.js

```diff
@@ -0,0 +1,39 @@
+/**
+* Copyright 2012-2017, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+/**
+ * Allow referencing a graph DOM element either directly
+ * or its id string
+ *
+ * @param {HTMLDivElement|string} gd: a graph element or its id
+ *
+ * @returns {HTMLDivElement} the DOM element of the graph
+ */
+// Get the container div: we store all variables for this plot as
+// properties of this div
+// some callers send this in by DOM element, others by id (string)
+module.exports = function(gd) {
```

Comment:
oops, this file was supposed to go in the next commit https://github.com/plotly/plotly.js/pull/2040/commits/aa434dd3b9b785dac6d37b24be2d89d1e52b8203

docs cleaned up in 0861736